hexagonal-architecture-employee-example/src/main/java/com/baeldung/architecture/hexagonal/employee/domain/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.architecture.hexagonal.employee.domain.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.baeldung.architecture.hexagonal.employee.domain.dto.Employee;
+import com.baeldung.architecture.hexagonal.employee.domain.exceptions.EmployeeException;
+import com.baeldung.architecture.hexagonal.employee.domain.repository.EmployeeRepository;
+
+@Component
+public class EmployeeServiceImpl implements EmployeeService {
+	@Autowired
+	private EmployeeRepository employeeRepository;
+	private int employeeId = 1;
```

Comment:
shouldn't be `static`

Reply:
employeeService being component, will be maintained as single instance in the JVM, hence need not be static.